src/profile/LanguageSelector.tsx

```diff
@@ -39,7 +39,7 @@ const LanguageSelector = (props: { user: User }) => {
   };
 
   return (
-    <Box>
+    <Box component="div">
```

Comment:
everything works here, but Im just curious about this component=div, I redid this one here and I dont remember seeing anything about this. Can you point me to where you found it?

Reply:
I got a bunch of red lines when I opened that file...see image
![image](https://user-images.githubusercontent.com/54069197/198868261-7f593401-49b6-46b5-9d1f-0cd27a0b1338.png)

I just looked up  the errror and found [this fix on stack overflow](https://stackoverflow.com/questions/68692230/ts-expression-produces-a-union-type-that-is-too-complex-to-represent-with-materi)

Reply:
Odd, but thank you for that, I think I can see why that would fix it.